guide/C/ch_basics.xml

```diff
@@ -625,144 +625,126 @@
-  				<entry align="center" namest="c4mysql"      > N </entry>
-  				<entry align="center" namest="c5postgresql" > N </entry>
-               </row>
-
-                <row>
-                  <entry >Save on command</entry>
-  				<entry align="center" namest="c2xml"        > Y </entry>
-  				<entry align="center" namest="c3sqlite"     > N </entry>
-  				<entry align="center" namest="c4mysql"      > N </entry>
-  				<entry align="center" namest="c5postgresql" > N </entry>
-                </row>
-
-                <row>
-                  <entry >Save on commit</entry>
-  				<entry align="center" namest="c2xml"        > N </entry>
-  				<entry align="center" namest="c3sqlite"     > Y </entry>
-  				<entry align="center" namest="c4mysql"      > Y </entry>
-  				<entry align="center" namest="c5postgresql" > Y </entry>
-                </row>
-
-                <row>
-                  <entry >Uses log files</entry>
-  				<entry align="center" namest="c2xml"        > Y </entry>
-  				<entry align="center" namest="c3sqlite"     > N </entry>
-  				<entry align="center" namest="c4mysql"      > N </entry>
-  				<entry align="center" namest="c5postgresql" > N </entry>
-                </row>
-
-                <row>
-                  <entry >Multi-user</entry>
+  				<entry align="center" namest="c5pgsql" > PostgreSQL </entry>
+			</row>
+
+			<row>
+                <entry namest="c1item"                      >Additional expertise</entry>
+  				<entry align="center" namest="c2xml" nameend="c3sqlite"  > None </entry>
+  				<entry align="center" namest="c4mysql" nameend="c5pgsql" > Database Administrator </entry>
+			</row>
+
+			<row>
+                <entry namest="c1item"                      >Compression</entry>
+  				<entry align="center" namest="c2xml"        > gzip </entry>
+  				<entry align="center" namest="c3sqlite" nameend="c5pgsql" > N/A </entry>
+			</row>
+
+			<row>
+                <entry namest="c1item"                      >File Save</entry>
+  				<entry align="center" namest="c2xml"        > On command </entry>
+  				<entry align="center" namest="c3sqlite" namend="c5pgsql" > On commit </entry>
+			</row>
+
+			<row>
+                <entry namest="c1item"                      >Multi-user</entry>
   				<entry align="center" namest="c2xml"        > N </entry>
   				<entry align="center" namest="c3sqlite"     > N </entry>
   				<entry align="center" namest="c4mysql"      > N </entry>
-  				<entry align="center" namest="c5postgresql" > N </entry>
-                </row>
-                </tbody>
-              </tgroup>
-            </table>
-			<para>1 - SQLite relies on libdbi. However, it is available by 
-			default on Mac OS and Windows. Linux users may need to manually 
-			install libdbi drivers for SQLite.</para>
-			<para>2 - MySQL and PostgreSQL place data within their own storage system.</para>
-		</sect3>
+  				<entry align="center" namest="c5pgsql" > N </entry>
```

Comment:
As this line is the last one using a single letter "N", how about writing it in full: "No". And optionally make it a horizontal span over all the columns. But feel free to judge that as you see fit.

Reply:
Did you miss this suggestion or did you deliberately choose not to implement it ? I didn't see any reaction from your side, hence my question.

Reply:
Geert, I did not see this. [Frankly, the conversation mode on GitHub falls apart when there are a lot of comments, as there are on this PR]. However, I am inclined to leave it as is. The point of including it here was to emphasize that multi-user is not at all viable at this time. Repeating it explicitly for each format underscores this. Saying "No" is minor, and is done.